hbase-server/src/main/java/org/apache/hadoop/hbase/tool/LoadIncrementalHFiles.java

```diff
@@ -803,14 +803,35 @@ private boolean checkHFilesCountPerRegionPerFamily(
    * <p>
    * Protected for testing.
    * @return empty list if success, list of items to retry on recoverable failure
+   * @deprecated as of release 2.3.0. Use {@link BulkLoadHFiles} instead.
```

Comment:
Will be removed as part of 3.0.0/4.0.0 ?

Reply:
Unnecessary, I think, because the whole class already has this comment.